sda-commons-server-auth/README.md

```diff
@@ -172,6 +172,18 @@ In addition, the client consumes the standard [proxy system properties](https://
 Please note that a specific proxy configuration in the `HttpClientConfiguration` disables the proxy system properties for the client using that configuration.
 This can be helpful when all clients in an Application should use the same proxy configuration (this includes all clients that are created by the [`sda-commons-client-jersey` bundle](../sda-commons-client-jersey).
 
+
+## Periodical reloading of public keys provided via JWKS
+
+Public keys provided via a [`JSON Web Key Set`](https://tools.ietf.org/html/draft-ietf-jose-json-web-key-41#section-5) (JWKS) are stored in a local key-cache.
+This is a potential security leak, because keys can be revoked by their issuer. The following scenario is possible: 
```

Comment:
Can we rephrase this text? A bit shorter and more on point. Focus on the behavior, not on the potential Problem. It could also tell (again) on how new keys are fetched. Something like:
* the first sentence with the key-cache
* for unknown keys, the cache is refreshed to search for the kid
* the key cache is refreshed periodically (each 5 min) to remove potentially revoked (I.e. removed from the key set) keys.
* If the key source is unavailable, the old keys stay valid (correct?) and no new kid's are loaded

Reply:
Actually I like to read about the problem that is solved to gain awareness about this security issue. But @dhenneke is right. The first part could be a bit shorter.

Reply:
I've shortened the description and put it right after the `Configuration`-part. Hopefully it's more concise now. Otherwise let me know!